CMakeLists.txt

```diff
@@ -62,6 +62,14 @@ else()
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++98")
 endif()
 
+if(NOT CXX11_ENABLED)
+  set(USE_LEGACY_IIDM ON CACHE BOOL "Force use of legacy IIDM library" FORCE)
```

Comment:
Why is this different from what is done in others side projects?
#libIIDM
if(NOT CXX11_ENABLED)
  set(USE_LEGACY_IIDM YES)
else()
  if(NOT MSVC)
    file(STRINGS ${DYNAWO_HOME}/dynawoEnv.txt DYNAWO_ENV REGEX "^[ ]*DYNAWO_USE_LEGACY_IIDM[ ]*=")
    string(REGEX REPLACE "^[ ]*DYNAWO_USE_LEGACY_IIDM[ ]*=[ ]*\"([^\"]*)\"" "\\1" USE_LEGACY_IIDM ${DYNAWO_ENV})
    unset(DYNAWO_ENV)
  endif()
endif()

Reply:
I used the same way as dynawo

Reply:
nevermind, there is a duplicata that I didn't see before